packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -139,6 +140,8 @@ export function generateGrid(
     const gqlDocuments: Record<string, string> = {};
     //const imports: Imports = [];
 
+    const fieldList = generateGqlFieldList({ columns: config.columns.filter((column) => column.name !== "id") }); // exclude id because it's always required
```

Comment:
I don't think that's correct. Sometimes we need fields that aren't in the grid but required for creation.

Reply:
this should and will be part of the pull-request fixing the copy/paste behaviour.